CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2021-09-08)
+
+- Added ability to place an element at the bottom of another element or before another element when using the dock service and the dynamic component service. [#250](https://github.com/blackbaud/skyux-core/pull/250)
```

Comment:
```suggestion
- Added the ability to place an element at the bottom of another element or before another element when using the dock service and the dynamic component service. [#250](https://github.com/blackbaud/skyux-core/pull/250)
```

Reply:
Also, `place an element at the bottom of another element or before another element` seems oddly worded to me, and I can't tell what we mean by "before." Could we change to `Added the place elements below or above other elements when...`? Or does that change the meaning?